<launch>
    <arg name="node_name" default="peter_chargingpile_search" />

    <node pkg="peter_chargingpile_search" type="search_chargingpile_node" name="$(arg node_name)" output="screen" >
    </node>

    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node">
        <remap from="turtle1/cmd_vel" to="ctrl_cmd_vel"/> 
    </node>



</launch>
